cli/cmd/plugin/unmanaged-cluster/log/log.go

```diff
@@ -13,13 +13,12 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/fatih/color"
 )
 
 const (
-	ColorNone       = "\033[0m"
-	ColorRed        = "\033[31m"
```

Comment:
Maybe I'm leaning towards over-engineering here, but would it make more sense to keep our Color constants here, but set their values to the library color values? That way if we every decide to replace `fatih/color` with something else, its usage isn't sprinkled all over our code base/

Reply:
While I don't feel too strongly either way, I do lean towards this as it would contain the `fatih/color` import to this package. Granted a lot of other packages will surely import this one :smiley: .

Reply:
I don't think this makes sense.

The interface that `fatih/color` gives us on individual `Color.Attributes` is a number of APIs (like `Sprint`, `Fprintf`, etc.) that write to IO writers, output a string, etc.

So if we went with something like
```go
const (
    ColorRed = Color.FgRed
    ....
)
```

further down, we would have to access the `fatih/color` Color.Attribute methods _anyways_ with something like
```
message := ColorRed.Sprint()
```

So the `ColorRed` indirection seems meaningless beyond us getting to define a single word variable; there would be no easy way to drop in and replace this library with a different one.

Reply:
Ah, I see what you mean. We would have to add a full abstraction layer over the top of `fatih/color` to isolate it. Not sure if that's worth it then. Probably fine to just use the import everywhere it's needed, and in the unlikely case that we decide to move to another library in the future just take care of the updates everywhere.

Reply:
It seems like a very lite weight library and doesn't come with many dependencies that will get in our way.

Thinking abit more about this, we do sort of already have a layer of abstraction in the `log` package where we have the various logging methods (`Warnf`, `Infof`, etc. etc.) and we could easily ad colors to that if end users of the log library didn't want to import the color library and use `log.Style` to dynamically style the colors of their logs.

Reply:
So maybe a _different_ approach I could take with this is to modify the various APIs in the `log` package we provide to use colors. And since we can be assured that they won't break in various different environments (like non-tty terminals, different color palettes, `NO_COLOR` scenarios, and even windows), end users of the library can use those without having to provide their own colors via `log.Style`